.github/workflows/release-bump-as-requirement.yml

```diff
@@ -0,0 +1,79 @@
+name: 'Release: Bump Action Scheduler version'
```

Comment:
Will this eventually become automatically run as part of the release?

If not, maybe it doesn't need to have the `Release:` prefix.

Reply:
> Will this eventually become automatically run as part of the release?

I was thinking it could be at least run periodically via cron so no one needs to actually trigger it (similar to bumping WP/PHP requirements). I agree that it probably doesn't need the `Release:` prefix.